src/main/java/io/swagger/codegen/v3/generators/DefaultCodegenConfig.java

```diff
@@ -4133,16 +4133,23 @@ else if (Parameter.StyleEnum.SPACEDELIMITED.equals(parameter.getStyle())) {
         }
     }
 
-    private boolean isObjectSchema (Schema schema) {
+    public boolean isObjectSchema (Schema schema) {
         if (schema instanceof ObjectSchema ||schema instanceof ComposedSchema) {
             return true;
         }
-        if (SchemaTypeUtil.OBJECT_TYPE.equals(schema.getType()) && !(schema instanceof MapSchema)) {
+        if (SchemaTypeUtil.OBJECT_TYPE.equalsIgnoreCase(schema.getType()) && !(schema instanceof MapSchema)) {
             return true;
         }
         if (schema.getType() == null && schema.getProperties() != null && !schema.getProperties().isEmpty()) {
             return true;
         }
+        if (StringUtils.isNoneEmpty(schema.get$ref())) {
```

Comment:
should be I guess `StringUtils.isNotBlank(..`

Reply:
right, was a kind of typo